Sources/TensorFlow/Layers/Cropping.swift

```diff
@@ -0,0 +1,86 @@
+// Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/// A layer for cropping tensors along the temporal dimension.
+public struct Cropping1D<Scalar: TensorFlowFloatingPoint>: Layer {
+    /// The cropping dimensions along the temporal axis.
+    @noDerivative public let cropping: (Int, Int)
+
+    /// Creates a cropping layer to trim the temporal dimension.
+    ///
+    /// - Parameter cropping: A tuple of two integers describing how many units should be trimmed
+    ///   off the beginning and end of the cropping dimension.
+    public init(cropping: (Int, Int) = (1, 1)) {
+        self.cropping = cropping
+    }
+
```

Comment:
@jon-tow for consistency of api, I think you should add symmetric initialization for the 1D layer as well.

Reply:
Yup. I wasn't sure about it but I'll add it in!